docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  app:
+    container_name: morse-code-ninja
+    build: .
+    volumes:
+      - .:/opt/morse-code-ninja/
+    environment:
+      - AWS_KEY_ID=${AWS_ACCESS_KEY_ID}
```

Comment:
I can update this to `AWS_ACCESS_KEY_ID` after https://github.com/zoglmannk/Morse-Code-Ninja/pull/45 is merged.